src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/cas2/ApplicationsController.kt

```diff
@@ -141,7 +140,7 @@ class ApplicationsController(
     .hmpps.approvedpremisesapi.jpa.entity.Cas2ApplicationSummary,
     user: NomisUserEntity,
   ):
-    ApplicationSummary {
+    uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.Cas2ApplicationSummary {
```

Comment:
minor: could this import path be moved to the top of the file? can see we have other models up there

Reply:
IntelliJ makes this an explicit import here because we already import the JPA Cas2ApplicationSummary there need it to avoid the ambiguity.  I've moved the import to the top and aliased model Cas2ApplicationSummary as `ModelCas2ApplicationSummary`, hopefully a reasonable compromise.